components/apimgt-extensions/org.wso2.carbon.apimgt.webapp.publisher/pom.xml

```diff
@@ -103,9 +103,11 @@
             <groupId>org.wso2.carbon.governance</groupId>
             <artifactId>org.wso2.carbon.governance.lcm</artifactId>
         </dependency>
+        <!-- californium dependancies -->
         <dependency>
-            <groupId>javax.ws.rs</groupId>
-            <artifactId>javax.ws.rs-api</artifactId>
+            <groupId>org.eclipse.californium</groupId>
+            <artifactId>cf-rd</artifactId>
+            <version>1.1.0-SNAPSHOT</version>
```

Comment:
californium resource directory is not in active development it seems[[1]](https://github.com/eclipse/californium.tools/graphs/contributors). We might need to consider our own implementation of this.